ui/pages/routes/routes.component.test.js

```diff
@@ -137,6 +144,111 @@ describe('Routes Component', () => {
       expect(getByTestId('account-menu-icon')).not.toBeDisabled();
     });
   });
+
+  describe('new network popup', () => {
+    const mockBtcAccount = createMockInternalAccount({
+      address: 'bc1q26a367uz34eg5mufwhlscwdcplu6frtgf00r7a',
+      name: 'Btc Account',
+      type: BtcAccountType.P2wpkh,
+      snapOptions: {
+        id: 'mock-btc-snap',
+        name: 'mock-btc-snap',
+        enabled: true,
+      },
+    });
+
+    const mockEvmAccount = createMockInternalAccount();
+
+    const mockNewlyAddedNetwork = {
+      chainId: CHAIN_IDS.BASE,
+      name: 'Base',
+      nativeCurrency: 'ETH',
+      defaultRpcEndpointIndex: 0,
+      rpcEndpoints: [
+        {
+          type: 'custom',
+          url: 'https://base.com',
+          networkClientId: CHAIN_IDS.BASE,
+        },
+      ],
+    };
+
+    const renderPopup = async (account) => {
+      process.env.IN_TEST = '';
```

Comment:
I guess the popup is not shown during tests? If yes, let's add a comment here:

```suggestion
      // This popup does not show up for tests, so we have to disable this:
      process.env.IN_TEST = '';
      
```